src/Components/profile/index.js

```diff
@@ -0,0 +1,216 @@
+  } else {
+    return (
+      <div className="profile-container">
+        <section className="user-name-profile-container">
+          <div className="avatar-name-profile-container">
+            <Avatar>{textAvatar}</Avatar>
+            <span className="user-name-text-p">{userName}</span>
+            <span
+              className="arrow-container-p"
+              onClick={() => {
+                setSeeModal(!seeModal);
+              }}
+            >
+              <ArrowDropDownIcon />
+            </span>
+          </div>
+        </section>
+        <div className="user-information-profile-container">
+          <Avatar className="user-avatar">{textAvatar}</Avatar>
+          <div className="user-information-p">
+            <h4 className="user-name-p">{nameUser}</h4>
+            <p className="email-user-p">{emailUser}</p>
+            <h4 className="user-name-p">{lastNameUser}</h4>
+          </div>
+        </div>
+        <div className="dashboard-view">
+          <span>My Events</span>
+          <span>
+            <ViewModuleIcon
+              className="viewModule-icon"
+              onClick={() => {
+                setViewEvents(true);
+              }}
+            />
+            <ViewStreamIcon
+              className="viewStream-icon"
+              onClick={() => {
+                setViewEvents(false);
+              }}
+            />
+          </span>
+        </div>
+        {}
+        <div
+          className={
+            viewEvents ? "dashboard-view-row" : "dashboard-view-column"
+          }
+        >
+          {currentEvents.map((event, index) => {
+            if (event.stateEvent === "EDIT" || event.stateEvent === "LEAVE") {
+              return (
+                <div
+                  key={event.id}
+                  className={
+                    viewEvents
+                      ? `element-${index} element`
+                      : `element-${index} element-column`
+                  }
+                  onClick={(e) => {
+                    showDetailEventClicked(
+                      e,
+                      setGoToDetailEvent,
+                      setEventClicked
+                    );
+                  }}
+                >
+                  <EventCard
+                    eventsList={eventsList}
+                    setEventList={setEventList}
+                    setGoToEditEvent={setGoToEditEvent}
+                    eventToEdit={eventToEdit}
+                    setEventToEdit={setEventToEdit}
+                    viewEvents={viewEvents}
+                    date={event.date}
+                    time={event.time}
+                    nameEvent={event.nameEvent}
+                    host={event.host}
+                    descriptionEvent={event.descriptionEvent}
+                    attendees={event.attendees}
+                    capacity={event.capacity}
+                    id={event.id}
+                    stateEvent={event.stateEvent}
```

Comment:
remove all these properties:
date={event.date}
                    time={event.time}
                    nameEvent={event.nameEvent}
                    host={event.host}
                    descriptionEvent={event.descriptionEvent}
                    attendees={event.attendees}
                    capacity={event.capacity}
                    id={event.id}
                    stateEvent={event.stateEvent}
and instead pass the event object to the component